codebase-stewardship/index.html

```diff
@@ -17,58 +17,58 @@
     <h1 class="title">Codebase stewardship</h1>
   </div>
   <div>
-    <h1>Everything your project needs to be sustainable, collaborative and open.</h1>
+    <h1>Everything your project needs to be sustainable, collaborative and trusted.</h1>
     <p>
-      Through <strong>codebase stewardship</strong> we help public-purpose open source codebases become successful by guaranteeing their quality and marketability. Trustworthy, usable, maintained and sustainable public code for your public organization.
+      Our codebase stewards help guarantee quality and marketability of public-purpose open source projects.
     </p>
   </div>
 </div>
 
 <img class="hero" alt="illustration, codebase stewardship" src="https://publiccodenet.github.io/illustrations/illustrations/codebase-2.svg">
 
 <div class="mp-section">
-  <div><h2>Making sure the code, tests and documentation are always up to the standards</h2>
+  <div><h2>Ensuring code, tests and documentation are up to standard</h2>
```

Comment:
This is an interesting one, both before and the new text. I believe we only ensure it's on par with the Standard for Public Code. We help checking towards other standards, but ensuring that compliance falls under the responsibility of the community.

Reply:
Do you think the new text need to specify the Standard for Public Code?

Reply:
Perhaps not. I guess "up to standard" here is an idiom rather than that we literally check the code to all standards. In that regard it is an improvement. However, is there a reason to not be explicit here?

Reply:
I will cut this knot in favour of the Standard for Public Code, as this is what we do now. We also explicitly do not check against other standards.